command/server/command.go

```diff
@@ -9,28 +9,193 @@ import (
 	"time"
 
 	"github.com/0xPolygon/minimal/minimal"
+	"github.com/0xPolygon/minimal/network"
+	"github.com/0xPolygon/minimal/types"
 	"github.com/hashicorp/go-hclog"
 	"github.com/mitchellh/cli"
 )
 
+// ServerFlagDescriptor contains the Description elements for a command flag. Duplicate of FlagDescriptor because of cyclic imports. Implements types.FlagDescriptor
+type ServerFlagDescriptor struct {
+	description       string   // Flag description
+	arguments         []string // Arguments list
+	argumentsOptional bool     // Flag indicating if flag arguments are optional
+	flagOptional      bool
+}
+
+func (s ServerFlagDescriptor) GetDescription() string {
+	return s.description
+}
+
+func (s ServerFlagDescriptor) GetArgumentsList() []string {
+	return s.arguments
+}
+
+func (s ServerFlagDescriptor) GetArgumentsOptional() bool {
+	return s.argumentsOptional
+}
+
+func (s ServerFlagDescriptor) GetFlagOptional() bool {
+	return s.flagOptional
+}
```

Comment:
We shouldn't have to create duplicate `structs` to resolve cyclic imports. 😐 
What do you think of creating a new `helper` package within `command` and move `MetaFlagDescriptor` there?

Reply:
You're completely right. I've moved it to a separate package, and now we have a single FlagDescriptor 👍